Anima/math-tools/multi_compartment_base/animaBaseCompartment.cxx

```diff
@@ -14,7 +14,11 @@ const double BaseCompartment::m_AzimuthAngleUpperBound = 2.0 * M_PI;
 const double BaseCompartment::m_DiffusivityUpperBound = 3e-3;
 const double BaseCompartment::m_RadialDiffusivityUpperBound = 1e-3;
 const double BaseCompartment::m_DefaultConcentrationUpperBound = 20;
+const double BaseCompartment::m_WatsonKappaUpperBound = 128.0;
 const double BaseCompartment::m_Epsilon = 1.0e-2;
+const double BaseCompartment::m_EAFLowerBound = 1.0 / 129.0;
```

Comment:
Why not 0 and 1 ? Or smaller value than roughly 10^-2 and 1.0 - 10^-2 ?

Reply:
A virer. J'utilise m_ZeroBound et m_FractionUpperBound pour l'IAF ou EAF.